lxd/device/disk.go

```diff
@@ -698,11 +707,24 @@ func (d *disk) startVM() (*deviceConfig.RunConfig, error) {
 						var errUnsupported UnsupportedError
 						if errors.As(err, &errUnsupported) {
 							d.logger.Warn("Unable to use virtio-fs for device, using 9p as a fallback", log.Ctx{"err": errUnsupported})
+
+							// Create a warning if virtiofsd is missing
+							if errUnsupported.Error() == "virtiofsd missing" {
```

Comment:
@monstermunchkin can you use a constant or sentinel error var for this rather than direct string comparison. Because if that returned string ever changes it'll break this functionality and likely the person changing it won't realise its used elsewhere. Using a constant or sentinel error var helps to tie these things together. Thanks